bundle/manifests/redhat-ods-operator-webhook-service_v1_service.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v1
```

Comment:
this file is not really needed in bundle.
https://github.com/red-hat-data-services/rhods-operator/pull/361

Reply:
@zdtsw I'm curious, Is there any specific issue if we keep this file ?